pages/landing/patcher/index.html

```diff
@@ -7,12 +7,15 @@
   - name: Use Cases
     url: /patcher/use-cases/
 
-title: "Keep your code up to date, even with breaking changes."
+extra_title: Patcher
+
+title: "Patcher | Gruntwork Labs"
+title_text: Keep your code up to date, even with breaking changes.
 excerpt: Patcher automatically creates pull requests to keep your dependencies up to date, updating not only the version numbers, but also patching your code to handle backwards incompatible changes.
 permalink: /patcher/
 
 hero:
-  image: /assets/img/infrastructure-cube-icons/infrastructure-in-a-day.png
+  image: /assets/img/landing-page-patcher/patcher.png
```

Comment:
Hm, this image ends up being a bit blurry, and perhaps even a little misleading. It makes Patcher seem like a purely CLI tool, whereas it's really more of a SaaS app that has a UI and opens PRs. Any other ideas for images?

Reply:
Hm, I was struggling to get anything else out of my hands yesterday..... I have another idea today, but let's see if I can make it work better. Thank you for the feedback - I think you're right, that it shows only the CLI side.